packages/dart/lib/src/hub_adapter.dart

```diff
@@ -221,4 +221,7 @@ class HubAdapter implements Hub {
 
   @override
   void removeAttribute(String key) => Sentry.currentHub.removeAttribute(key);
+
+  @override
+  void captureSpan(Span span) => Sentry.currentHub.captureSpan(span);
```

Comment:
I'm not sure about the naming here. Can you point me to what `captureSpan` is supposed to do?